src/utils/deskStructure.js

```diff
@@ -22,11 +22,16 @@ export const uawitStructure = (S) =>
         .icon(ControlsIcon)
         .child(
           S.list()
-            .title('Configuration')
+            .title('Settings')
             .items([
-              ...S.documentTypeListItems().filter((listItem) =>
-                ['widget'].includes(listItem.getId())
-              ),
+              S.listItem()
+                .title('General Configuration')
+                .child(
+                  S.document()
+                    .title('General Configuration')
+                    .schemaType('settings')
+                    .documentId('general')
+                )
```

Comment:
I like this!!!

`S.document()
     .title('General Configuration')
     .schemaType('settings')
     .documentId('general')`
It shows on the right panel the document of Settings (our schema) with predefined Id (general)

So, it is possible to have several documents of Settings with predefined Ids. And content manager can edit only predefined amount of items but cannot create new ones.